core/arch/arm/plat-stm32mp2/main.c

```diff
@@ -180,3 +182,27 @@ void plat_external_abort_handler(struct abort_info *ai __unused)
 	/* External abort may be due to SERC events */
 	stm32_serc_handle_ilac();
 }
+
+void __noreturn do_reset(const char *str __maybe_unused)
+{
+	struct rstctrl *rstctrl = NULL;
+
+	if (CFG_TEE_CORE_NB_CORE > 1) {
+		/* Rely on multi_core_halt_it_handler() to trap CPUs */
```

Comment:
I think you should not use the function label here, in case it changes in the future.
`/* Halt execution of other CPUs */` should be enough.